docs/template_plugin/src/template_async_infer_request.cpp

```diff
@@ -21,16 +20,16 @@ TemplateAsyncInferRequest::TemplateAsyncInferRequest(
     _inferRequest(inferRequest), _waitExecutor(waitExecutor) {
     _pipeline = {
         {cpuTaskExecutor, [this] {
-            IE_PROFILING_AUTO_SCOPE(PreprocessingAndStartPipeline)
+            OV_ITT_SCOPED_TASK(itt::domains::TemplatePlugin, "PreprocessingAndStartPipeline");
```

Comment:
Without quotes is looks better.

Reply:
In some places string with spaces is used. Therefore I can't define macro for argument without quotes but I'll define macro without arguments where function name is inserted automatically.

Reply:
@vladislav-volkov Anyway we should keep possibility specify name if we want to trace some region.